snow/consensus/snowball/binary_snowball_test.go

```diff
@@ -109,53 +81,39 @@ func TestBinarySnowballAcceptWeirdColor(t *testing.T) {
 	sb := binarySnowball{}
 	sb.Initialize(beta, red)
 
-	if pref := sb.Preference(); pref != red {
-		t.Fatalf("Wrong preference. Expected %d got %d", red, pref)
-	} else if sb.Finalized() {
-		t.Fatalf("Finalized too early")
-	}
+	require.Equal(red, sb.Preference())
+	require.False(sb.Finalized())
 
 	sb.RecordSuccessfulPoll(red)
 	sb.RecordUnsuccessfulPoll()
 
-	if pref := sb.Preference(); pref != red {
-		t.Fatalf("Wrong preference. Expected %d got %d", red, pref)
-	} else if sb.Finalized() {
-		t.Fatalf("Finalized too early")
-	}
+	require.Equal(red, sb.Preference())
+	require.False(sb.Finalized())
 
 	sb.RecordSuccessfulPoll(red)
+
```

Comment:
nit: remove to reduce diffs

Reply:
These gaps were done on purpose, I think it reads better